conf/suse/master

```diff
@@ -537,18 +537,24 @@ syndic_user: salt
 # The renderer to use on the minions to render the state data
 #renderer: yaml_jinja
 
-# The Jinja renderer can strip extra carriage returns and whitespace
-# See http://jinja.pocoo.org/docs/api/#high-level-api
-#
-# If this is set to True the first newline after a Jinja block is removed
-# (block, not variable tag!). Defaults to False, corresponds to the Jinja
-# environment init variable "trim_blocks".
-#jinja_trim_blocks: False
-#
-# If this is set to True leading spaces and tabs are stripped from the start
-# of a line to a block. Defaults to False, corresponds to the Jinja
-# environment init variable "lstrip_blocks".
-#jinja_lstrip_blocks: False
+# TODO documentation
+# Ignoring documentation for now
+# (until agreement on implementation has been reached)
+
+# Default jinja environment options for everything but
+# salt sls files
+#jinja_env:
+#  line_statement_prefix: '#'
```

Comment:
make this synced with the documentation below.
I vote for ``%`` rather than ``#`` or ``''``